# generated by patch-package 6.5.0 on 2022-04-04 19:10:51
#
# command:
#   npx patch-package key-did-provider-ed25519
#
# declared package:
#   key-did-provider-ed25519: 1.1.0
#
diff --git a/node_modules/key-did-provider-ed25519/dist/key-did-provider-ed25519.cjs.development.js b/node_modules/key-did-provider-ed25519/dist/key-did-provider-ed25519.cjs.development.js
index 5649b95..210778b 100644
--- a/node_modules/key-did-provider-ed25519/dist/key-did-provider-ed25519.cjs.development.js
+++ b/node_modules/key-did-provider-ed25519/dist/key-did-provider-ed25519.cjs.development.js
@@ -480,13 +480,13 @@ var runtime = (function (exports) {
   // iterator prototype chain incorrectly implement this, causing the Generator
   // object to not be returned from this call. This ensures that doesn't happen.
   // See https://github.com/facebook/regenerator/issues/274 for more details.
-  Gp[iteratorSymbol] = function() {
+  define(Gp, iteratorSymbol, function() {
     return this;
-  };
+  });
     
-  Gp.toString = function() {
+  define(Gp, "toString", function() {
     return "[object Generator]";
-  };
+  });
 
   function pushTryEntry(locs) {
     var entry = { tryLoc: locs[0] };
diff --git a/node_modules/key-did-provider-ed25519/dist/key-did-provider-ed25519.esm.js b/node_modules/key-did-provider-ed25519/dist/key-did-provider-ed25519.esm.js
index 74bad77..e5612c8 100644
--- a/node_modules/key-did-provider-ed25519/dist/key-did-provider-ed25519.esm.js
+++ b/node_modules/key-did-provider-ed25519/dist/key-did-provider-ed25519.esm.js
@@ -474,13 +474,14 @@ var runtime = (function (exports) {
   // iterator prototype chain incorrectly implement this, causing the Generator
   // object to not be returned from this call. This ensures that doesn't happen.
   // See https://github.com/facebook/regenerator/issues/274 for more details.
-  Gp[iteratorSymbol] = function() {
+  define(Gp, iteratorSymbol, function() {
     return this;
-  };
+  });
     
-  Gp.toString = function() {
+  define(Gp, "toString", function() {
     return "[object Generator]";
-  };
+  });
+
 
   function pushTryEntry(locs) {
     var entry = { tryLoc: locs[0] };
